cabal-testsuite/PackageTests/Backpack/Includes2/setup-internal-fail.cabal.out

```diff
@@ -12,5 +12,6 @@ Error:
                            brought into scope by mixins: fail:mysql (Database.MySQL as Database)
                         or 'fail-0.1.0.0-postgresql:Database.PostgreSQL'
                            brought into scope by mixins: fail:postgresql (Database.PostgreSQL as Database)
-      While filling requirements of build-depends: fail:mylib
+      While filling requirements of     build-depends: fail:mylib
+                                    and build-depends: fail:mylib
```

Comment:
@ezyang

Any pointers on fixing the duplicate output? I think it may be some only-main-lib code I didn't delete when adding multilib handling. It does not affect functionality